py1

```diff
@@ -0,0 +1,82 @@
+string = '''                   CHAPTER I
```

Comment:
Could you read this string for a file instead of "hard-coding" it into the file? That way I can run the same exercise with a different book